### DISCUSSION TITLE
[Feature] set album cover from mobile

### DISCUSSION THREAD

**Discussion by waclaw66** [immich-app/immich#3789] [Feature Request]
### The feature

Currently there is not possible to set album cover from mobile.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by ArtemSmaznov** [comment]
I also find it looking for it from time to time on mobile.
When (hopefully not 'if') implemented, should ideally be in an individual picture's options, but having it in album's drop-down is nice as well.

**Reply by Howy75** [reply]
Still not implemented, as I see.

**Comment by timonrieger** [comment]
#25515
